src/Helpers_TEST.cc

```diff
@@ -963,21 +967,41 @@ TEST(HelpersTest, AppendToStream)
 {
   std::ostringstream out;
 
-  math::appendToStream(out, 0.12345678, 3);
```

Comment:
Can you keep these tests and use `IGN_UTILS_WARN_IGNORE__DEPRECATED_DECLARATION` and `IGN_UTILS_WARN_RESUME__DEPRECATED_DECLARATION`. We can remove the test when we remove the function.

Reply:
c35a039b2c4b33867d46380784a3ba70b801d1c9